src/workerd/server/tests/python/BUILD.bazel

```diff
@@ -66,3 +66,12 @@ py_wd_test(
         "0.27.1",
     ],
 )
+
+py_wd_test(
+    "dont-snapshot-pyodide",
+    # TODO: Requires a new bundle deploy
+    skip_python_flags = [
+        "0.26.0a2",
```

Comment:
Does this still require a new bundle? Seems like by moving the logic to pyodide.c++ you can enable these variants

Reply:
No.